data_rows/LoadHistory.js

```diff
@@ -0,0 +1,42 @@
+import DataRow from "./DataRow";
+import { TYPES } from "./Types";
+
+/**
+ * LOAD_HISTORY View
+ *
+ * https://docs.snowflake.net/manuals/sql-reference/account-usage/load_history.html
+ */
+export class LoadHistory extends DataRow {
+  constructor(tableID, date) {
+    super();
+    this.TABLE_ID = tableID;
+    this.LAST_LOAD_TIME = date.toISOString();
+    this.ROW_COUNT = Math.round(Math.random() * 100);
+    this.ERROR_COUNT = Math.random() < 0.1 ? Math.round(Math.random() * 10) : 0;
+  }
+
+  static oddsNew() {
```

Comment:
personally I would define this as a const at the top of the module, `const ODDS_NEW 0.2` but I don't really have any good reason to do it one way or the other.

Reply:
Early on I thought I'd have a lot more "dice rolling" to create an object than I ended up with. So I made this `DataRow.rollDice()` / `SubClass.oddsNew()` construct that ended up not being used very much (and actually `DataRow.oddsNew()` should be `static`). What I'd like is class level properties but doesn't seem like those exist in JS?

I could set `const ODDS_NEW = 0.2` at the module level and then reference it in the `static oddsNew()` method in each class to help with the "magic number" issue, is that what you're referring to? or would you do the check differently?